exodus/search/templates/search_form.html

```diff
@@ -1,107 +1,165 @@
 {% load i18n %}
```

Comment:
`search_form.html` sounds strange for me as it's the whole home page. What do you think? Is it something complicated to change?

Reply:
I am actually not sure the "search" module altogether makes sense (it's mostly empty except for this template). I don't know if I want to change it in this PR, but it could be worth moving this into another module.